src/components/Table/Table/__stories__/table.stories.mdx

```diff
@@ -0,0 +1,187 @@
+                <TableCell>{rowItem.sentOn}</TableCell>
+                <TableCell>{rowItem.subject}</TableCell>
+                <TableCell>
+                  <TableAvatar text={rowItem.sentBy} />
+                </TableCell>
+                <TableCell>
+                  <Label text={rowItem.status} isAnimationDisabled />
+                </TableCell>
+                <TableCell>{rowItem.emailsSent}</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      );
+    }};
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ Table } />
+
+## Loading
+Using skeleton
+<Canvas>
+  <Story name="Loading">
+    {() => {
+      return (
+        <Table
+          dataState={{ isError: false, hasMoreResults: false }}
+          errorState={<TableErrorState />}
+          emptyState={<TableEmptyState />}
+          columns={emailColumns}
+        >
+          <TableHeader>
+            {emailColumns.map((headerCell, index) => (
+              <TableHeaderCell
+                key={index}
+                title={headerCell.title}
+              ></TableHeaderCell>
+            ))}
+          </TableHeader>
+          <TableBody>
+            {[...new Array(5)].map((key, index) => (
+              <TableRow key={key}>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.TEXT} height={20} width={index % 2 === 0 ? 140 : 70} />
+                </TableCell>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.TEXT} height={20} width={index % 2 === 0 ? 200 : 120} />
+                </TableCell>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.CIRCLE} height={24} width={24} />
+                </TableCell>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.TEXT} height={20} />
+                </TableCell>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.TEXT} height={20} />
+                </TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      );
+    }};
+  </Story>
+</Canvas>
+
+## Empty State
+Table has no data to show
+<Canvas>
+  <Story name="Empty State">
+    <Table {...basicTableProps} emptyState={<TableEmptyState />}>
+      <TableHeader>
+        <TableHeaderCell title="First Column" />
+        <TableHeaderCell title="Second Column" />
+      </TableHeader>
+      <TableBody>
+        {/* Empty */}
+      </TableBody>
+    </Table>
+  </Story>
+</Canvas>
+
+## Scroll
+This is an example of a table with 5000 rows
+<Canvas>
+  <Story name="Scroll">
+    {() => {
+      const Row = ({ id, num, text }, index, style) => {
+        return (
+          <TableRow key={id} style={style}>
+            <TableCell>{num}</TableCell>
+            <TableCell>{`${text} ${text} ${text} ${text} ${text} ${text} ${text} ${text} ${text} ${text} ${text}`}</TableCell>
+            <TableCell>1</TableCell>
```

Comment:
Using hard coded examples for the row can be confusing because it's not the common use case when implementing virtualized tables (or tables in general)

Also since Row is the only accepted children for virtualized body, why can't we receive TableRows as children, and id always  should be key...maybe the renderer function can return only the table row children?